### PR TITLE
Add support for the Huion new 1060 plus 2048.

### PR DESCRIPTION
**Pull request by jamesbt365**
Verification:
https://canary.discord.com/channels/615607687467761684/615611007951306863/909609554482958377
https://canary.discord.com/channels/615607687467761684/615611007951306863/909613160141574205

### CODE REVIEW

File: OpenTabletDriver.Configurations/Configurations/Huion/new 1060 plus 2048.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "Name": "Huion new 1060 plus (2048)",
```
Comment (X9VoiD) [inline OpenTabletDriver.Configurations/Configurations/Huion/new 1060 plus 2048.json:2]:
```suggestion
  "Name": "Huion New 1060 Plus (2048)",
```
and rename the file to `New 1060 Plus (2048)`.

File: OpenTabletDriver.Configurations/Configurations/Huion/new 1060 plus 2048.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  "Specifications": {
+    "Digitizer": {
+      "Width": 254.0,
+      "Height": 158.75,
+      "MaxX": 40000.0,
+      "MaxY": 25000.0
```
Comment (X9VoiD) [inline OpenTabletDriver.Configurations/Configurations/Huion/new 1060 plus 2048.json:8]: What LPI is this tablet? Verify that the width and height is physically correct.

**Review by X9VoiD — CHANGES_REQUESTED** [review]

File: OpenTabletDriver.Configurations/Configurations/Huion/New 1060 Plus (2048).json [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{
+  "Name": "Huion New 1060 plus (2048)",
```
Comment (X9VoiD) [inline OpenTabletDriver.Configurations/Configurations/Huion/New 1060 Plus (2048).json:2]: Big Plus

Comment (jamesbt365) [reply OpenTabletDriver.Configurations/Configurations/Huion/New 1060 Plus (2048).json]: corrected.

**Review by X9VoiD — APPROVED** [review]

**Pull request merged**